introduction.md

```diff
@@ -1,9 +1,7 @@
-* This Is Shubham
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?